packages/notifications/resources/views/notifications.blade.php

```diff
@@ -1,6 +1,6 @@
 <div
     @class([
-        'filament-notifications pointer-events-none fixed inset-4 z-50 mx-auto flex justify-end gap-3',
+        'pointer-events-none fixed inset-4 z-50 mx-auto flex justify-end gap-3 filament-notifications',
```

Comment:
Custom classes should be at the start according to Tailwind's ordering convention.

Reply:
Ah.